src/theme/components/template.ts

```diff
@@ -44,13 +44,14 @@ const TemplateContent = defineStyleConfig({
     rowGap: "grid.l",
   }),
   // With left or right sidebars, we need to set two grid columns and
-  // the column for the sidebar is max 255px width.
+  // the column for the sidebar is max 271px width (255px for the sidebar,
+  // + 16px for padding).
```

Comment:
Thank you for updating this! As I was on the bus home last night, I was thinking that the comment should be updated to explain the oddball number (271px).